doc/manual/manual.bib

```diff
@@ -11270,15 +11270,17 @@ @Article{rose_freesurface
 }
 
 @article{heister_aspect_methods2,
-  Title                    = {High Accuracy Mantle Convection Simulation through Modern Numerical Methods. {II}: {R}ealistic Models and Problems},
-  Author                   = {Heister, Timo and Dannberg, Juliane and Gassm{\"o}ller, Rene and Bangerth, Wolfgang},
-  Journal                  = {Geophysical Journal International},
-  Year                     = {2017},
-  Number                   = {2},
-  Pages                    = {833-851},
-  Volume                   = {210},
-  Doi                      = {10.1093/gji/ggx195},
-  Url                      = {https://doi.org/10.1093/gji/ggx195}
+  title                    = {High Accuracy Mantle Convection Simulation through Modern Numerical Methods.
```

Comment:
The article above (rose_freesurface uses the same capitalization. Can you unify that as well?